packages/client/src/components/SafeHome.js

```diff
@@ -3,24 +3,11 @@ import SafeOverview from "./SafeOverview";
 import ThingsToDo from "./ThingsToDo";
 import TransactionHistory from "./TransactionHistory";
 
-const SafeHome = ({
-  safeData,
-  allBalance,
-  allNFTs,
-  actions,
-  address,
-  onSign,
-  onConfirm,
-}) => (
+const SafeHome = (props) => (
   <>
-    <SafeOverview allBalance={allBalance} allNFTs={allNFTs} />
-    <ThingsToDo
-      actions={actions}
-      onSign={onSign}
-      onConfirm={onConfirm}
-      safeData={safeData}
-    />
-    <TransactionHistory safeData={safeData} address={address} />
+    <SafeOverview {...props} />
+    <ThingsToDo {...props} />
```

Comment:
`ThingsToDo` was missing `onReject` and `onApprove` so wasnt able to take those actions on items. since `SafeHome` is just a wrapper component that bundles other components to render, lets just spread all props to its children and children can decide what to use

Reply:
Pasting it here again in case you missed it:
I would suggest against passing props to all children
1. It creates redundant dependencies, causing components to be dependent and to re-render on irrelevant state changes.
2. It hurts readability, in order to figure out what's been passed to the component, another developer has to to a lot of tracking and code tracing.

onReject and onApprove is missing because of a bad merge conflict, creating ways to accommodate it will lead to more error in the future

Reply:
just trying to reduce boilerplate and places that need to be updated, but can update to xenias solution